csrc/scheduler/mma_utils.h

```diff
@@ -370,6 +370,13 @@ int64_t computeExpectedSharedMemoryUsage(
 //!  Z = complex<double>
 char dtypeToChar(const DataType& dtype);
 
+//! Check if the inner ID of maybeAllocationDomain of the consumer when
+//! mapped to the consumer is the inner ID of the consumer's
+//! maybeAllocationDomain. A typical use of this function would be to help
+//! determine if a tranposed ldMatrix is required. If this function returns
```

Comment:
The "A typical use.." sentence in this comment is probably no longer needed, since the new function name makes that clear.